src/TConsole.cpp

```diff
@@ -777,7 +777,7 @@ void TConsole::closeEvent( QCloseEvent *event )
 
     if( profile_name != "default_host" && ! mUserAgreedToCloseConsole )
     {
-        ASK: int choice = QMessageBox::question( this, "Exiting Session: Question", "Do you want to save the profile "+profile_name, QMessageBox::Yes|QMessageBox::No|QMessageBox::Cancel );
+        ASK: int choice = QMessageBox::question( this, tr("Save profile?"), tr("Do you want to save the profile %1?").arg(profile_name), QMessageBox::Yes|QMessageBox::No|QMessageBox::Cancel );
```

Comment:
Of course it also becomes an inverted question mark at the start for Spanish at least 😆 

Having different behaviour for similar constructs is inconsistent IMVHO - if we have to put a question mark on one the other ought to have the period/full-stop for consistency.  OTOH Given that the other message is a `QMessageBox::Critical` an exclamation mark at the end (¡With an inverted one at the beginning for the Spanish translation! 😁) may not be unreasonable...